include/pbconvert/sm.hpp

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef PBCONVERT_SM_HPP_
+#define PBCONVERT_SM_HPP_
+
+#include <aos/common/cloudprotocol/alerts.hpp>
+#include <aos/common/cloudprotocol/envvars.hpp>
+#include <aos/common/cloudprotocol/log.hpp>
+#include <aos/common/monitoring/monitoring.hpp>
+#include <aos/common/types.hpp>
+#include <aos/sm/networkmanager.hpp>
+
+#include <servicemanager/v4/servicemanager.grpc.pb.h>
+
+namespace aos::common::pbconvert::sm {
+/**
+ * Converts aos push log to protobuf.
+ *
+ * @param src push log to convert.
+ * @param[out] dst protobuf log data.
+ * @return void.
+ */
+void ConvertToProto(const cloudprotocol::PushLog& src, ::servicemanager::v4::LogData& dst);
```

Comment:
Here as well some API, returns value through param ref some by return. Could we have same pattern for all? Either return all in return or all by ref. Also mixed pattern is possible for example use return for basic types and ref for struct classes etc.